src/Oro/Bundle/DataGridBundle/Extension/Pager/Orm/Pager.php

```diff
@@ -139,24 +138,28 @@ public function init()
 
         /** @var QueryBuilder $query */
         $query = $this->getQueryBuilder();
+        $countQb = $this->countQueryBuilderOptimizer->getCountQueryBuilder($this->getQueryBuilder());
 
         $query->setFirstResult(null);
         $query->setMaxResults(null);
 
+
         if (count($this->getParameters()) > 0) {
             $query->setParameters($this->getParameters());
```

Comment:
Are you happy for me to bind them to query before line [141](https://github.com/LRotherfield/platform/blob/hotfix/datagrid-complex-join-pagination/src/Oro/Bundle/DataGridBundle/Extension/Pager/Orm/Pager.php#L141) so that the clone on line [60](https://github.com/LRotherfield/platform/blob/hotfix/datagrid-complex-join-pagination/src/Oro/Bundle/BatchBundle/ORM/QueryBuilder/CountQueryBuilderOptimizer.php#L60) of the countQueryBuilder gets them?